lib/zlib.js

```diff
@@ -430,6 +430,11 @@ Zlib.prototype.close = function close(callback) {
   this.destroy();
 };
 
+Zlib.prototype._destroy = function _destroy(err, callback) {
+  _close(this);
+  callback(err);
+};
```

Comment:
Does this mean that we can set `Zlib.prototype.close = Zlib.prototype.destroy;`?

Reply:
I tried doing that but it made a lot of the sync apis throw assertion errors because of some timing issues i guess